operate/importer/src/main/java/io/camunda/operate/zeebeimport/RecordsReaderHolder.java

```diff
@@ -75,9 +79,32 @@ public boolean hasPartitionCompletedImporting(final int partitionId) {
     return partitionsCompletedImporting.contains(partitionId);
   }
 
-  public int incrementEmptyBatches(final int partitionId, final ImportValueType importValueType) {
+  public void incrementEmptyBatches(final int partitionId, final ImportValueType importValueType) {
     final var reader = getRecordsReader(partitionId, importValueType);
-    return countEmptyBatchesAfterImportingDone.merge(reader, 1, Integer::sum);
+    countEmptyBatchesAfterImportingDone.merge(reader, 1, Integer::sum);
```

Comment:
💭 Just thinking out loud:
Could there ever be a race condition while incrementing this emptyBatches value?
I mean, could it happen that two different threads are ran against the same reader and they come to increment this value in the same time? 
if it's a possibility, then we should make the value here an AtomicInteger. But I'm not sure if this is really an issue to worry about.